mobile/src/main/java/org/openhab/habdroid/util/Util.java

```diff
@@ -66,6 +66,38 @@ public static String normalizeUrl(String sourceUrl) {
         return normalizedUrl;
     }
 
+    private static List<OpenHABSitemap> removeDefaultFromSitemapList(String defaultName, List<OpenHABSitemap> sitemapList) {
+
+        // Only default sitemap is given, so don't remove it
+        if (sitemapList.size() == 1) {
+            return sitemapList;
+        }
+
+        List<OpenHABSitemap> sitemapListResult = new ArrayList<OpenHABSitemap>();
+
+        // Remove default sitemap
+        for (int i = 0; i < sitemapList.size(); i++) {
+            if (sitemapList.get(i).getName().equals(defaultName)) {
+                continue;
+            }
+            sitemapListResult.add(sitemapList.get(i));
+        }
+
+        return sitemapListResult;
+    }
+
+    private static List<OpenHABSitemap> sortSitemapList(List<OpenHABSitemap> sitemapList) {
+        // Sort by sitename name
+        Collections.sort(sitemapList, new Comparator<OpenHABSitemap>() {
+            @Override
+            public int compare(OpenHABSitemap sitemap1, OpenHABSitemap sitemap2) {
+                return  sitemap1.getName().compareTo(sitemap2.getName());
```

Comment:
Sitemaps have the attribute "Label", so please use that to compare

Reply:
Hey @mueller-ma 

Ok, next time I use a descripive name :) sorry. And one fix per pull.

1) I think that my change is a little more elegant then the fix implemented in #578. I remove the default sitemap only if one sitemap is configured on the server. And I use only one method (no copy and paste).

2) Yes sitemap has a attribute label. But some times a user want to sort the sitemap list by e.g. "priority". So the user can use
     Filename/Name -> Label 
     1Livinroom -> Livingroom
     2Bathroom -> Bathroom
     ...

If name=label (happens frequently) -> works too

Should I really go back to Label?

3) I run the App without a sitemap, one sitemap and ten sitemaps. No error. Which class performs the test?

Thanks,
Mario

Reply:
> next time I use a descripive name 

I prefere this time :) You can edit PR names.

> Yes sitemap has a attribute label. But some times a user want to sort the sitemap list by e.g. "priority"

We should check how basic ui sorts sitemaps.

> I run the App without a sitemap, one sitemap and ten sitemaps. No error. Which class performs the test?

When you run the app with one ui, no selection popup is shown, but the test expects a popup.